.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+version: 2.1
+orbs:
+  aws-s3: circleci/aws-s3@1.0.0
+  node: circleci/node@2.0.2
+jobs:
+    build:
+        docker:
+            - image: circleci/python:3.7
+        parameters:
+            aws_bucket:
+                type: string
+                default: "${AWS_BUCKET}"
+        steps:
+            - when:
+                condition: <<parameters.aws_bucket>>
+                steps:
+                    - checkout
+                    - node/install:
+                        install-yarn: false
+                        node-version: latest
+                    - create_zip_upload_to_s3:
+                        lambdaFolder: "cloud-watch-to-slack-testing-58006136-c30b-43d1-880c-7b36f860e7c8/deployment"
```

Comment:
jw why are all these folder names so weird?

Reply:
Not entirely sure, I think they have relevance on AWS.